src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -105,15 +105,16 @@ public function iProceedSelectingOfflinePaymentMethod($paymentMethodName)
 
     /**
      * @When /^I proceed selecting "([^"]+)" shipping method$/
+     * @When /^I proceed selecting "([^"]*)" shipping method and "([^"]*)" as shipping country$/
      * @Given /^I chose "([^"]*)" shipping method$/
      */
-    public function iProceedSelectingShippingMethod($shippingMethodName)
+    public function iProceedSelectingShippingMethodAndShippingCountry($shippingMethodName, $shippingCountry = null)
     {
         $this->checkoutAddressingStep->open();
         $this->checkoutAddressingStep->fillAddressingDetails([
             'firstName' => 'John',
             'lastName' => 'Doe',
-            'country' => 'France',
+            'country' => $shippingCountry ?: 'France',
```

Comment:
Later we should refactor to reflect following logic: If country is not specified, select first available. We should avoid hardcoding France, especially that we removed this "single France channel ..." thingy. :)